cmd/boulder-wfe2/main.go

```diff
@@ -111,10 +97,7 @@ type Config struct {
 		// by that intermediate. The first chain representing any given issuing
 		// key pair will be the default for that issuer, served if the client does
 		// not request a specific chain.
-		// NOTE: This config field deprecates the CertificateChains and
-		// AlternateCertificateChains fields. If it is present, those fields are
-		// ignored. They will be removed in a future release.
-		Chains [][]string `validate:"required_without=CertificateChains,omitempty,min=1,dive,min=2,dive,required"`
+		Chains [][]string `validate:"required,min=1,dive,min=2,dive,required"`
```

Comment:
I'm a bit confused about the double required. The config validation for this is now `chains` is required and `chains[[]]` is required?

Example from our config validation docs:
```
{
  "chains": [
    ["bar", "baz"],
    ["baz", "bar"],
  ],
}
```

Reply:
Outer `required`: means that ([][]string) cannot be `nil`.
Outer `min=1`: means that the length of outer slice must be >=1.
Inner `min=2`: means that the length of each of the inner slice must contain >=2.
Inner `required`: means that the each of the strings in the inner slice cannot be "".